templates/stamp/clone-github-repository.sh

```diff
@@ -2,6 +2,8 @@
 #
 # Copyright (c) Microsoft Corporation. All Rights Reserved.
 # Licensed under the MIT license. See LICENSE file on the project webpage for details.
+#
+# todo: is this dead code or is it invoked through arm template's extension scripts? (aka oxa-tools/templates/stamp/stamp*json files)
```

Comment:
We may still need this for future iterations especially with regards to installing monitoring bits